sequelizeDatabase/sequelFunctions.js

```diff
@@ -264,54 +265,78 @@ insertFromCsvLine = (
     Comments,
     "\n"
   );
-  insertPerson(FundID, EmployeeName, LinkedInUrl, Comments)
-    .then(newPerson => {
-      insertCompany(PortfolioCompanyName, FundID).then(originalCompany => {
-        insertOriginalFundPosition(newPerson.IndividualID, originalCompany.CompanyID, OriginalPostion)
-          .then(() => {
-            //NOTE: I'm hoping it doesn't freak out that I included no arguements here.
-            insertEmployeeHistory(
-              UserID,
-              newPerson.IndividualID,
-              originalCompany.CompanyID,
-              OriginalPostion,
-              OriginalStartDate,
-              OriginalEndDate
-            ).then(() => {
-              insertCompany(CurrentEmployer, FundID).then(newCompany => {
-                insertEmployeeHistory(
-                  UserID,
-                  newPerson.IndividualID,
-                  newCompany.CompanyID,
-                  CurrentPostion,
-                  OriginalStartDate,
-                  OriginalEndDate
-                );
+  //returns IndividualID for use in processing quarter updates
+  return new Promise((resolve, reject) => {
+    insertPerson(FundID, EmployeeName, LinkedInUrl, Comments)
+      .then((newPerson) => {
+        insertCompany(PortfolioCompanyName, FundID).then((originalCompany) => {
+          insertOriginalFundPosition(newPerson.IndividualID, originalCompany.CompanyID, OriginalPostion)
+            .then(() => {
+              //NOTE: I'm hoping it doesn't freak out that I included no arguements here.
+              insertEmployeeHistory(
+                UserID,
+                newPerson.IndividualID,
+                originalCompany.CompanyID,
+                OriginalPostion,
+                OriginalStartDate,
+                OriginalEndDate
+              ).then(() => {
+                insertCompany(CurrentEmployer, FundID).then((newCompany) => {
+                  insertEmployeeHistory(
+                    UserID,
+                    newPerson.IndividualID,
+                    newCompany.CompanyID,
+                    CurrentPostion,
+                    OriginalStartDate,
+                    OriginalEndDate
```

Comment:
**IMPORTANT:** Here's a problem from when I first made this function; Since I didn't know what date to use, I used the _same_ dates for both the original fund position, _and_ for the new position. While that isn't part of this branch, it's an issue that will need to be corrected/updated soon.